app/uk/gov/hmrc/gform/gform/SectionRenderingService.scala

```diff
@@ -1703,6 +1710,11 @@ class SectionRenderingService(frontendAppConfig: FrontendAppConfig, lookupRegist
     }
   }
 
+  private def dataLabelAttribute(label: SmartString): Map[String, String] =
+    dataLabelAttribute(label.localised.value(LangADT.En))
+  private def dataLabelAttribute(label: String): Map[String, String] =
+    Map("data-label" -> label.replaceAll("''", "'"))
```

Comment:
What is the need to replace quotes like this?

Reply:
To unescape `'`:
https://tonnygaric.com/blog/escape-single-quotes-when-using-java-messageformat